capture/generate/qrandom.py

```diff
@@ -348,9 +344,12 @@ def wolfram_sampling(expoverview, rdict, vollimits, rxndict, wellnum, userlimits
     # how long can a fix remain this hot?
     # todo calculate the joules in this fix
     # We're serving fixes hot, boys, lets let this one cool down a bit
-    portion_df['Reagent6 (ul)'] = np.floor(portion_df['Reagent7 (ul)'] / 2).astype(int)
-    portion_df['Reagent7 (ul)'] = np.ceil(portion_df['Reagent7 (ul)'] / 2).astype(int)
-    rdict['6'] = rdict['7']
+    if rxndict['ExpWorkflowVer'] < 1.1:
+        if rxndict['lab'] == 'LBL':
```

Comment:
This is a very good change. Even better would be documenting what each of the workflows mean to our different scientists and developers and document this specific behavior for LBL along with its rationale. If we don't do this it will be very easy for it to slip our minds.

Reply:
https://drive.google.com/open?id=19CTUNiA4TAH00L8c_iEx5zE0k9pVH11p

Here is a start.  I have the conversation pinned and will start writing up documentation somewhere soon. I just haven't been willing to spend time on documentation over writing papers yet.  

I will pull together stuff and see how far things get.